app/views/organizations/show.html.erb

```diff
@@ -1,100 +1,128 @@
+  <% end %>
 
-<% if @organization.image3.present? %>
-    <%= image_tag @organization.image3.url(:small) %>
+  <% if @organization.image3.present? %>
+  <%= image_tag @organization.image3.url(:small) %>
   <% end %>
 
-<% if @organization.image4.present? %>
-  <%= image_tag @organization.image4.url(:small) %>
-<% end %>
+  <% if @organization.image4.present? %>
+    <%= image_tag @organization.image4.url(:small) %>
+  <% end %>
 
 
-<% if can? :manage, @organization %>
-  <% if @organization.published %>
-    <p>Published</p>
+  <% if can? :manage, @organization %>
+    <% if @organization.published %>
+      <p>Published</p>
+    <% end %>
   <% end %>
-<% end %>
 
 
-<% if can? :cru, @organization %>
-  <%= link_to "Edit", edit_organization_path(@organization), class: "btn btn-primary" %>
-<% end %>
-
+  <% if can? :cru, @organization %>
+    <%= link_to "Edit", edit_organization_path(@organization), class: "btn btn-primary" %>
+  <% end %>
 
-<% if !(@claimed && @claimed.status?) || @organization.user.nil?  %>
-  <%= link_to "Claim this business", new_organization_claim_request_path(@organization), class: "btn btn-primary"%>
-<% end %>
 
-<hr>
-<div class="container-fluid col-lg-6 col-md-6 col-sm-6 col-xs-12" id="twitter-feed">
-  <% if @organization.twitter  %>
-    <a class="twitter-timeline" href="https://twitter.com/<%= @organization.twitter %>" data-screen-name="<%= @organization.twitter %>">Tweets by @<%= @organization.twitter %></a>
-    <script>!function(d,s,id){var js,fjs=d.getElementsByTagName(s)[0],p=/^http:/.test(d.location)?'http':'https';if(!d.getElementById(id)){js=d.createElement(s);js.id=id;js.src=p+"://platform.twitter.com/widgets.js";fjs.parentNode.insertBefore(js,fjs);}}(document,"script","twitter-wjs");twttr.widgets.load();</script>
+  <% if !(@claimed && @claimed.status?) || @organization.user.nil?  %>
+    <%= link_to "Claim this business", new_organization_claim_request_path(@organization), class: "btn btn-primary"%>
   <% end %>
+
+  <div class="map-container" style='width: 100%;'>
+    <div id="map-org" class="single-map" style='width: 100%; height: 400px'></div>
+  </div>
+
+  <script type="text/javascript">
+    function initialize() {
+      var orgLocation = {lat: <%= @organization.latitude %>, lng: <%= @organization.longitude %>};
+      // Create a map object and specify the DOM element for display.
+      var map = new google.maps.Map(document.getElementById('map-org'), {
+        center: orgLocation,
+        scrollwheel: false,
+        zoom: 13
+      });
+      // Create a marker and set its position.
+      var marker = new google.maps.Marker({
+        map: map,
+        position: orgLocation,
+        title: '<%= @organization.name %>'
+      });
+      // Content of info window
+      // var companyname = <%= @organization.name %>;
+      // var companyoverview = <%= @organization.overview %>;
+
+      // Create an info window for the marker
+      var infowindow = new google.maps.InfoWindow({
```

Comment:
Use camelcase with js variables names. Write `infoWindow` instead of `infowindow`